RELEASE/scripts/autoscend/auto_equipment.ash

```diff
@@ -339,6 +339,10 @@ void finalizeMaximize()
 	{
 		addBonusToMaximize($item[familiar scrapbook], 200); // scrap generation for banish/exp
 	}
+	if(in_wildfire() && auto_haveFireExtinguisher())
+	{
+		addBonusToMaximize($item[industrial fire extinguisher], 200); // extinguisher prevents per-round hot damage in wildfire path 
+	}
```

Comment:
I'd suggest doing this in pre-adv if(place.fire_level > 3)

Reply:
good idea, did it